apps/frontend-web/app/test/[id]/component/top-bar.tsx

```diff
@@ -0,0 +1,29 @@
+import { Button } from '@test-assessment/ui-components';
+import React from 'react';
+
+const PageTopBar = ({ handleClick, setShowModal }) => {
```

Comment:
should have handleViewClick, handleDeleteClick, handleDuplicateClick, handleInviteClick

Reply:
props are just placeholder